virtualenv.py

```diff
@@ -1593,10 +1607,12 @@ def resolve_interpreter(exe):
     if os.path.abspath(exe) != exe:
         paths = os.environ.get('PATH', '').split(os.pathsep)
         for path in paths:
-            if os.path.exists(os.path.join(path, exe)):
-                exe = os.path.join(path, exe)
+            check_exe = resolve_exe(os.path.join(path, exe))
+            if check_exe:
+                exe = check_exe
                 break
-    if not os.path.exists(exe):
+    exe = resolve_exe(exe)
+    if not exe:
         logger.fatal('The executable %s (from --python=%s) does not exist' % (exe, exe))
```

Comment:
By overwriting ```exe``` here you've made the error message misleading (as it no longer states the same argument as the user supplied).